mapping-for-navigation/_posts/0000-01-09-modeling-intersections-for-map-navigation.md

```diff
@@ -179,13 +183,15 @@ Try to avoide creating sausage roads.
 | [![][dcb8bcea]][dcb8bcea] | [![][e5c6b7e5]][e5c6b7e5] |
 | (_[fullsize][dcb8bcea]_ \| _[wireframe][fd01f060] view_) | (_[fullsize][e5c6b7e5]_ \| _[wireframe][1ac0124a] view_)  |
 
-  [dcb8bcea]: ../../images/intersection-modeling/4way_3dual_carriageways_and_bidir.png "(fullsize view)"
-  [e5c6b7e5]: ../../images/intersection-modeling/4way_3dual_carriageways_and_bidir_err.png "(fullsize view)"
-  [fd01f060]: ../../images/intersection-modeling/4way_3dual_carriageways_and_bidir_wireframe.png "(wireframe view)"
-  [1ac0124a]: ../../images/intersection-modeling/4way_3dual_carriageways_and_bidir_wireframe_err.png "(wireframe view)"
+  [dcb8bcea]: ../../images/intersection-modeling/4way_3dual_carriageways_and_bidir.png "Intersection of 3 dual carriageways and 1 bi-drectional road (fullsize view)"
+  [e5c6b7e5]: ../../images/intersection-modeling/4way_3dual_carriageways_and_bidir_err.png "Incorrect drawing of the intersection of 3 dual carriageways and 1 bi-drectional road (fullsize view)"
+  [fd01f060]: ../../images/intersection-modeling/4way_3dual_carriageways_and_bidir_wireframe.png "Intersection of 3 dual carriageways and 1 bi-drectional road (wireframe view)"
+  [1ac0124a]: ../../images/intersection-modeling/4way_3dual_carriageways_and_bidir_wireframe_err.png "Incorrect drawing of the intersection of 3 dual carriageways and 1 bi-drectional road (wireframe view)"
 
 -   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=19/33.70308/-117.86370) 🗺
 
-In this case, it is need to keep a balance between cartography and navigation approach of mapping. This case as a gold point between them that helps to keep visual and algoritmical simplicity of the data. (_Based on the [discussion][6a7f9f11] in the US-Talk mailing list_)
+In this case, it is important to keep a balance between cartography and navigation approach to mapping. This method works as a golden mean between both and helps to maintaine visual and algoritmical simplicity of the data. (_Based on the [discussion][6a7f9f11] in the US-Talk mailing list_)
```

Comment:
Typos:
s/maintaine/maintain/
s/algoritmical/algorithmic/ (I assume this is referring to routing algorithms not needing extra turn restrictions?)